node/tests/integration/main.rs

```diff
@@ -486,14 +573,27 @@ async fn assert_slot_status(
     );
 }
 
+async fn assert_slot_status_call(
+    client: &AzeClient,
+    account_id: AccountId,
+    slot_data: GameStorageSlotData
+) {
+    let (account, _) = client.get_account(account_id).unwrap();
+    let game_account_storage = account.storage();
+
+    let remaining_balance = slot_data.player_balance() - slot_data.highest_bet();
+    let slot_index = 68;

```

Comment:
Let's do one thing let's add these slot related constants in `lib/src/constant/mod.rs` with appropriate constants name so that these values became declarative.